### DISCUSSION TITLE
alternate delta display

### DISCUSSION THREAD

**Discussion by seiyria** [NightscoutFoundation/xDrip#3106] [Ideas]
Hey all, I'm a Diabox refugee, recently moved to Juggluco/xDrip. One of my favorite features was the delta display in diabox. I see xdrip has a similar display, but there is a difference. Given the example of changing +10 over 5 minutes:

- in Diabox, the delta would be displayed as a delta per timeframe, so it would display +2/min
- in xdrip, the delta seems to be displayed per timeframe, so it would display +10

Is there a way to get it to display in the style of the former rather than the latter? I find it's a lot easier for me to recognize problems when I see +5/min rather than +25.

**Comment by Navid200** [comment]
I wonder why this was never done the way you describe as a slope instead of a delta.

@tzachi-dar Any thoughts?
